### PR TITLE
Bazelrific

### PR DESCRIPTION
**Pull request by sorensenjs**
Adding BUILD files across the tree, this is a big change and will likely break the pipeline
for the short term. I'm running it again on latin and may push some more changes outside
of unit test lang.

### CODE REVIEW

**Comment by sorensenjs** [comment]
Fudge, this isn't working due to pickling problems for the packages.

**Comment by sorensenjs** [comment]
Ugh, does cloud dataflow even work with Bazel? Table flip.

File: wikiconv/.bazelrc [hunk 1]
```diff
@@ -1,6 +1,5 @@
-tartup --host_jvm_args=-Xmx2500m
+startup --host_jvm_args=-Xmx2500m
 startup --host_jvm_args=-Xms2500m
```
Comment (iislucas) [inline wikiconv/.bazelrc]: I'm a bit confused that you need exactly the same line twice in the `.bazelrc`, if it's needed, would be good to have a comment as to why.

Comment (sorensenjs) [reply wikiconv/.bazelrc]: These are not the exact same line, Xms is the starting size, Xmx is the maximum size.
Arguably these parameters are unnecessary, but this is just correcting the typo at the beginning of the file.

**Comment by sorensenjs** [comment]
Successful end-to-end run for latin.

**Pull request merged**